docs/src/python/user-guide/misc/visualization.py

```diff
@@ -0,0 +1,141 @@
+# --8<-- [start:setup]
+import base64
+
+import polars as pl
+
+url = "docs/data/iris.csv"
+
+# --8<-- [end:setup]
+
+# --8<-- [start:dataframe]
+
+iris_agg_df = (
+    pl.scan_csv(url)
+    .group_by("species")
+    .agg(
+        pl.col("petal_length").mean(),
+    )
+    .collect()
+)
+
+# --8<-- [end:dataframe]
+
+# --8<-- [start:hvplot_make_plot]
+import hvplot
+
+plot = iris_agg_df.plot.bar(
+    x="species",
+    y="petal_length",
+    width=650,
+)
+hvplot.save(plot, "docs/images/hvplot_bar.html")
+with open("docs/images/hvplot_bar.html", "r") as f:
+    chart_html = f.read()
+    print(f"{chart_html}")
+
+# --8<-- [end:hvplot_make_plot]
+"""
+# --8<-- [start:hvplot_show_plot]
```

Comment:
I trieed building this locally but don't see it

![image](https://github.com/pola-rs/polars/assets/33491632/805c6d49-e58a-4d9d-bd94-c145b448656a)

does it work for you?

Reply:
> does it work for you?

Yes, they all work for me. I'll look into it